scripts/rff/historical_weather_variability/county/R/heatwave_5year_rasters.R

```diff
@@ -0,0 +1,119 @@
+rm(list=setdiff(ls(), c("master_path", "code_path")))
+
+
+path <- master_path 
+setwd(path)
+
+# count 
+load(paste0(path, "replication/data/heatwaves/hw_count_dataframe.Rdata"))
+ref <- raster(paste0(path, "replication/data/heatwaves/4km/annual/hw_count_1981.tif"))
+
+data$count_1981_1985 = data$hw_count_1981 + data$hw_count_1982 + 

```

Comment:
Might be easier to abstract these away in a function and access the column names programmatically. In R, I believe one can assign column name strings to variables and then access them: https://stackoverflow.com/questions/19730806/access-data-frame-column-using-variable

Reply:
Marking resolved to submit.